lib/pdk/validators/ruby/rubocop.rb

```diff
@@ -16,6 +16,10 @@ def self.cmd
         'rubocop'
       end
 
+      def self.pattern
+        ['**/*.rb']
```

Comment:
I recall we had a brief discussion about how rubocop should lint anything that has ruby in it, including Gemfile and Rakefile. Adding a pattern will ignore those files now, is that desired behavior?